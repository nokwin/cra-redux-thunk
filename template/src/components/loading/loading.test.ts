import { Loading } from "./loading.component";

describe("Loading component", () => {
  it("should render loading component", () => {
    expect(Loading).toMatchSnapshot();
  });
});
