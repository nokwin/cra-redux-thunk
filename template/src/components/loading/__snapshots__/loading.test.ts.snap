// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render loading component 1`] = `[Function]`;
